lib/_stream_writable.js

```diff
@@ -558,7 +558,11 @@ function clearBuffer(stream, state) {
 }
 
 Writable.prototype._write = function(chunk, encoding, cb) {
-  cb(new ERR_METHOD_NOT_IMPLEMENTED('_write()'));
+  if (this._writev) {
+    this._writev([{ chunk, encoding }], cb);
```

Comment:
I guess this introduces a small performance penalty but the user can implement `_write()` to avoid it.